client/app/controllers/index.js

```diff
@@ -4,6 +4,11 @@ export default Ember.Controller.extend({
     newCollectionTitle: '',
     modelCache : null,
     filterText : '',
+    currentPage: 1,
+    loadingMore: false,
+    showLoadMore: Ember.computed('model.meta', function(){
+        return this.get('model.meta.pagination.count') > this.get('model.length') ? true : false;
```

Comment:
You should be able to just return `this.get('model.meta.pagination.count') > this.get('model.length') `